package.json

```diff
@@ -57,6 +57,8 @@
     "ole": "^0.0.3"
   },
   "dependencies": {
+    "babel-polyfill": "^6.26.0",
+    "flush-promises": "^1.0.2",
```

Comment:
Do we really need these 2 modules ?

Reply:
Aimed to flush promises to test Promise more easily in jest.